### ISSUE TITLE
How to structure data using feed-me + JSON + Hyper?

### ISSUE THREAD

**Issue by DavidStrada** [verbb/hyper#64]
### Question

Using the following throws an error:
```
"websiteUrl": {
        "type":"verbb\\hyper\\links\\Url",
        "handle": "default-verbb-hyper-links-url",
        "linkValue": "www.google.com",
        "linkText": "some text"
    },
```

### Additional context

I get the following error.

```
Too few arguments to function craft\feedme\helpers\DataHelper::fetchValue(), 2 passed in /vendor/verbb/hyper/src/integrations/feedme/fields/Hyper.php on line 47 and exactly 3 expected - DataHelper.php: 122
```

**Comment by engram-design** [comment]
Can you let me know what versions of everything you're using? The below should be all you require. Note the `type` is (for the moment) a [shortened string](https://github.com/verbb/hyper/blob/9b7fa1394b9029252dd682896012b71c3bbbfde4/src/integrations/feedme/fields/Hyper.php#L51-L61), and not the full class:

```json
"websiteUrl": {
    "type":"url",
    "linkValue": "www.google.com",
    "linkText": "some text"
}
```

**Comment by DavidStrada** [comment]
@engram-design sure, here's current setup
```
"craftcms/cms": "^4.4.7", 
"craftcms/feed-me": "5.1.1",
 "verbb/hyper": "^1.1",
```

Tested your example and it fails with the same message

```
Too few arguments to function craft\feedme\helpers\DataHelper::fetchValue(), 2 passed in /vendor/verbb/hyper/src/integrations/feedme/fields/Hyper.php on line 47 and exactly 3 expected - DataHelper.php: 122
```


thanks for your help.

**Comment by DavidStrada** [comment]
@engram-design in [here](https://github.com/verbb/hyper/blob/craft-4/src/integrations/feedme/fields/Hyper.php#L47) if we pass the obj. it stops failing `$preppedData[$subFieldHandle] = DataHelper::fetchValue($this->feedData, $subFieldInfo, $this->feed);` it works.

**Comment by engram-design** [comment]
This is because you're on an older version of Feed Me. Refer to this [fix](https://github.com/craftcms/feed-me/blob/develop/CHANGELOG.md#5111---2023-03-24). I'd recommend updating Feed Me, but happy to make that change here.

**Comment by engram-design** [comment]
Fixed in [1.1.7](https://github.com/verbb/hyper/releases/tag/1.1.7)
